tests/protractor/e2e/contacts/add-new-health-center.js

```diff
@@ -0,0 +1,43 @@
+const commonElements = require('../../page-objects/common/common.po.js'),
+	contactPage = require('../../page-objects/contacts/contacts.po.js'),
+	helper = require('../../helper'),
+	utils = require('../../utils');
+
+describe('Add new health center tests : ', () => {
+	afterEach(done => {
+		utils.resetBrowser();
```

Comment:
This should not be necessary and it does a browser refresh which is really slow. Why do we need to refresh?